### DISCUSSION TITLE
Transparency for latex concealed items.?

### DISCUSSION THREAD

**Discussion by semi710** [tiagovla/tokyodark.nvim#39] [Q&A] [answered]
![image](https://github.com/tiagovla/tokyodark.nvim/assets/60490474/9fbbefb5-7f7b-442c-bba2-d7b2ce651c3b)
Here `x^2` is concealed by latex ig. but it does not have a transparent bg How can i fix that?

**Comment by tiagovla** [comment, marked as answer]
Changing the highlight group `Conceal` should do it. If you update your tokyodark.nvim, it should respect the `transparent_background` option.
